Makefile.am

```diff
@@ -207,4 +207,7 @@ merge-clean:
 	@find ./ | $(GREP) \.OTHER | xargs rm -f
 	@find ./ | $(GREP) \.BASE | xargs rm -f
 	@find ./ | $(GREP) \~$$ | xargs rm -f
-	
+
+dist-hook:
+	cp $(distdir)/wolfssl/options.h.in $(distdir)/wolfssl/options.h
```

Comment:
Why would we distribute a template to a dist dir?

Reply:
We have a step in the release procedure where we copy the options.h template over the options.h. I am automating that step with this change when building the dist or dist-zip targets.

options.h is updated by the configure script based on the users configure enables, so the template needs to be in the distdir.

When we forget to or don't follow that copy step in the release procedure, IDE builds like Windows will try to build with my last configure options.

Reply:
Thanks!